server/sockets.js

```diff
@@ -446,6 +446,29 @@ if (cluster.isMaster) {
 		}
 	}, 10 * MINUTES);
 
+	/**
+	 * @param {string} message
+	 * @param {number} channelid
+	 */
+	const extractChannel = (message, channelid) => {
```

Comment:
Apparently you can't use `function extractChannel` if you want to make it `const`. I still think the `function` notation looks better.

Reply:
It's at least more consistent with the rest of the codebase. If we want to break with established style, we should probably do it all at once.

Reply:
lol i had gone with `const` over `function` because literrally everything else in this block is a `const`, not a` function`. also, IIRC functions names and definitions gets hoisted in JS, so declaring them in a block is sort of misleading.

Reply:
Functions are apparently block-scoped in ES2015 (previously, functions in if-blocks were undefined behavior).

But you're right, this is fine and should be left alone. Matching the other code in this file is better. Although probably some sort of restructure so we're not defining functions in if-blocks is probably the _best_ approach here.